# Project 1

> formal BI procedure

1. get clean reads from biocompany
2. turn fastq file into fasta file
  - **galaxy project**
  - **awk**
  - Biopython
3. format db 
  - use makeblast db now  (**note on evernote**)
  - ncbi recommend to use datebase packaged with that kind of file instead of make that file from fasta
4. lcoal blast
  - follow the formal procedure
5. caculate the total number of CDS/gene
  -  BI-I-2. Linux
     -   awk grep sort uniq | combined together
